make/test.mk

```diff
@@ -437,5 +437,13 @@ display-eval:
 
 .PHONY: test
 ## Run the unit tests in the 'testsupport/...' packages
-test:
+test: test-support test-setup
+	 @echo 'Run unit tests in test-support and setup'
```

Comment:
```suggestion
```
The output actually happens after the tests are executed so might as well remove it

Reply:
done in [ea2ffc7](https://github.com/codeready-toolchain/toolchain-e2e/pull/1038/commits/ea2ffc75f4cc97ce2ad76ebf424647ad8ad67b91)